packages/sui-studio-utils/src/domain-builder/index.js

```diff
@@ -61,10 +62,18 @@ export default class DomainBuilder {
                 return self._useCases[useCase].success !== undefined
                   ? Promise.resolve(
                       typeof successResponse === 'function'
-                        ? successResponse(params)
+                        ? inlineError
+                          ? [null, successResponse(params)]
+                          : successResponse(params)
+                        : inlineError
+                        ? [null, successResponse]
                         : successResponse
                     )
-                  : Promise.reject(self._useCases[useCase].fail)
+                  : Promise.resolve(
```

Comment:
👀 This is a breaking change for people not using inlineError.

Reply:
That suppose to be a ternary too.

But it is true in this way it is a  breaking change.